quickjs-libc.c

```diff
@@ -479,7 +488,54 @@ typedef JSModuleDef *(JSInitModuleFunc)(JSContext *ctx,
                                         const char *module_name);
 
 
-#if defined(_WIN32) || defined(__wasi__)
+#if defined(_WIN32)
+static JSModuleDef *js_module_loader_so(JSContext *ctx,
+                                        const char *module_name)
+{
+    JSModuleDef *m;
+    HINSTANCE hd;
+    JSInitModuleFunc *init;
+    char *filename = NULL;
+    int len = strlen(module_name);
+    if (len > 2 &&
+        ((module_name[0] >= 'A' && module_name[0] <= 'Z') ||
+            (module_name[0] >= 'a' && module_name[0] <= 'z')) &&
+        module_name[1] == ':') {
+        filename = (char *)module_name;
+    } else if (len > 2 && module_name[0] != '.' && (module_name[1]!= '/' || module_name[1] == '\\')) {
+        filename = js_malloc(ctx, strlen(module_name) + 2 + 1);
+        if (!filename)
+            return NULL;
+        strcpy(filename, "./");
+        strcpy(filename + 2, module_name);
+    }
+    {
+        hd = LoadLibraryA(filename);
+        if (filename!= module_name)
+            js_free(ctx, filename);
+        if (hd == NULL) {
+            JS_ThrowReferenceError(ctx, "js_load_module '%s' error: %lu",
+                                module_name, GetLastError());
```

Comment:
```suggestion
                                    module_name, GetLastError());
```
I believe that lines up the arguments but please check.